### DISCUSSION TITLE
UX/UI: Reorder "Stores" List - Default Store

### DISCUSSION THREAD

**Discussion by astupidmoose** [btcpayserver/btcpayserver#3515] [Ideas / Feature Requests]
Sorry if this already exists somewhere but I have never really found this option. 

I'd like to see some way of reordering the stores list in the drop down menu. At the moment I have a specific store that in my # 1 slot that is mainly unused, which means everytime I open btcpay, I need to select the store I typically use most which is like # 5 slot. 

I would love to be able to reorder these based on my needs.

**Comment by Kukks** [comment]
I wonder if the new view solves it since the last used store saved and
loaded by default now

**Reply by dennisreimann** [reply]
The user preference settings and selected store ID get removed on log out though. Perhaps that's involved here?

**Reply by astupidmoose** [reply]
Yes, on logout it clears the presences from what I can tell. Having the option of Manually setting store order would solve this.

**Comment by Goro2030** [comment]
It will be great also if the stores would be sorted alphabetically in the dropdown list to quickly find what you're looking for.

**Reply by dennisreimann** [reply]
Done in #3599
